{{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: postgres
-        image: docker-registry.default.svc:5000/openshift/postgresql:9.6
+        image: docker.io/bitnami/postgresql:9.6
```

Comment:
Is there a specific reason why we stick to version 9.6? Could we move on to version 12 instead? There are a few worthwhile new features in Postgres version>10.

### References

- [PostgreSQL 10 release notes](https://www.postgresql.org/docs/10/release-10.html)
- [PostgreSQL 11 release notes](https://www.postgresql.org/docs/11/release-11.html)
- [PostgreSQL 12 release notes](https://www.postgresql.org/docs/12/release-12.html)
- [Postgres versioning policy](https://www.postgresql.org/support/versioning/)

Reply:
No, no reason.  I just used the version that was already in the file.  Yes, we should try out updating the version.